src/ansys/sherlock/core/stackup.py

```diff
@@ -823,16 +831,16 @@ def get_layer_count(self, project, cca_name):
             raise e
 
     @require_version()
-    def get_stackup_props(self, project, cca_name):
+    def get_stackup_props(self, project: str, cca_name: str) -> StackupProperties:
         """Get the stackup properties from a CCA.
 
         Available Since: 2021R2
 
         Parameters
         ----------
-        project : str, required
+        project: str

```

Comment:
The convention we have established is to note optional parameters. It is assumed that a parameter is required if it isn't documented as optional. It is even more obvious with type hints.